client/src/components/ConfirmPaymentButton/ConfirmPaymentButton.tsx

```diff
@@ -0,0 +1,40 @@
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { confirmPayments } from '../../helpers/APICalls/confirmPayments';
+import { useHistory } from 'react-router-dom';
+
+interface Props {
+  totalCost: number;
+}
+
+export default function AddCard({ totalCost }: Props): JSX.Element {
+  const { updateSnackBarMessage } = useSnackBar();
+  const { loggedInUser, loggedInUserProfile, fetchProfileAndUpdateContext } = useAuth();
+  const history = useHistory();
+
+  const handleSubmit = async () => {
```

Comment:
Some of the code in this PR does not belong to this ticket, please be mindful when making PRs :)